src/Context/LoginContext.php

```diff
@@ -315,14 +316,21 @@ protected function generateMember($email, $password, $group, $identifier)
 
         // make sure any validation for password is skipped, since we're not testing complexity here
         $validator = Member::password_validator();
-        Member::set_password_validator(null);
+        $nullValidator = new PasswordValidator();
+        $nullValidator->setMinLength(0);
+        $nullValidator->setMinTestScore(0);
+        $nullValidator->setHistoricCount(0);
+        Member::set_password_validator($nullValidator);
+
+        // Write member record
         $member->FirstName = $identifier;
         $member->Surname = "User";
         $member->Email = $email;
-        $member->PasswordEncryption = "none";
```

Comment:
This is the main change for this PR - anything else is done to support this change.